pytorch_lightning/trainer/trainer.py

```diff
@@ -167,7 +169,12 @@ def __init__(
 
             check_val_every_n_epoch: Check val every n train epochs.
 
-            fast_dev_run: runs 1 batch of train, test  and val to find any bugs (ie: a sort of unit test).
+            fast_dev_run:
+                .. warning:: .. deprecated:: 0.7.2
+
```

Comment:
```suggestion
```

Reply:
I see a blank line after each of the deprecated parameters. I thought it was a standard practice Should i remove the blank line after?

Reply:
@Borda The blank line is needed, otherwise sphinx will complain with a warning: "explicit markup ends without a blank line". Believe me or not it was a pain to find that out when I removed the warnings from the docs xD

Reply:
In that case, i will drop the recommendation. Feel free to suggest any more changes. If there is no other change, PR is ready to be merged